### PR TITLE
Do not repeat blame parent links if present

### LINKED ISSUE #143: Blame parent links could be duplicated when navigating back
Follow-up to #24, similar to #114 and #135.

screen shot 2016-04-01 at 22 11 31

It's a bit strange only the first one is repeated.

### PR DESCRIPTION
**Pull request by hkdobrev**
Fixes #143

### CODE REVIEW

File: extension/content.js [hunk 1]
```diff
@@ -189,17 +189,23 @@ function infinitelyMore() {
 function addBlameParentLinks() {
 	$('.blame-sha').each((index, commitLink) => {
```
Comment (paulmolluzzo) [inline extension/content.js]: You need to make sure you're not selecting the links you just shoved into the DOM:
```js
$('.blame-sha').not('.js-blame-parent').each((index, commitLink) => {
```

Without this fix there's an error:
```
Uncaught TypeError: Cannot read property '0' of null                             content.js:198
```

**Comment by paulmolluzzo** [comment]
With the fix I suggested above it's working nicely, so if you want to do that I'll give it a whirl and a thumbs up.

Good catch here! Took me a few times to even reproduce it.

will-smith-shahrukh-khan-fresh-prince-c1OleUXbLc3cs

**Comment by sindresorhus** [comment]
What direction are we going with on this?

**Comment by paulmolluzzo** [comment]
Waiting on the fix I suggested. There's an error without it.

**Comment by hkdobrev** [comment]
I will fix it, I just wasn't much available the last few days as you've seen.

**Comment by sindresorhus** [comment]
Ok, no worries. Was just wondering where we were at on this one :)

**Comment by hkdobrev** [comment]
Updated. @paulmolluzzo

**Comment by paulmolluzzo** [comment]
LGTM :shipit:

**Pull request merged**